Resources/doc/cookbook/recipe_custom_action.rst

```diff
@@ -92,7 +92,10 @@ to implement a ``clone`` action.
 
     class CRUDController extends Controller
     {
-        public function cloneAction()
+        /**
+         * @param $id
+         */
+        public function cloneAction($id)
```

Comment:
IIRC there might be a third option by calling a method of the admin class whose exact name I forgot